docs/data/versions.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This format looks really nice! I like this a lot!

Just leaving this comment as a reminder: we'll want to remove all of the 2.0-related entries here before merging. At some point in the future, when we're ready to publish our 2.0 docs, we can re-add those members and everything should automatically work.

Reply:
Removed